### DISCUSSION TITLE
HTTPS proxy (apache) tries Connecting and results in Not Ready

### DISCUSSION THREAD

**Discussion by GusevVictor** [rustdesk/rustdesk#9722] [General]
### Bug Description

Dear all I've setup apache proxy server and after that rustdesk client cant connect to hbbr/hbbs server.

Just Connecting to the rustdesk network.... State

### How to Reproduce

Install apache like this one:

```
<VirtualHost *:443>
SSLEngine on
SSLCertificateFile /etc/letsencrypt/live/xxx/fullchain.pem
SSLCertificateKeyFile /etc/letsencrypt/live/xxx/privkey.pem
ProxyRequests On
ProxyVia On
AllowCONNECT 4443 21114-21119
ErrorLog ${APACHE_LOG_DIR}/error_forward_proxy.log
CustomLog ${APACHE_LOG_DIR}/access_forward_proxy.log combined
</VirtualHost>
```

### Expected Behavior

Success connection state but I see in apache log only 21116 port connect as OK. There is NO any attempt to connect to ports like 21114, 21115 (very rarely), 21117, 21118, 21118

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

![image](https://github.com/user-attachments/assets/eefbb1b8-df12-42c9-8cd6-bac3d655d4b8)

**Comment by Poordeveloper** [comment]
https://github.com/rustdesk/rustdesk/discussions/5687#discussioncomment-9232011

**Reply by GusevVictor** [reply]
Thanks a lot for your answer. Will it be the same problem with udp for a sockcs proxy setup? I've tested it with dante proxy with no luck also. I use free version.
